src/AbstractLinesBeforeNamespaceFixer.php

```diff
@@ -51,6 +55,8 @@ protected function fixLinesBeforeNamespace(Tokens $tokens, $index, $expected)
             } elseif (substr_count($previous->getContent(), "\n") !== $expected) {
                 $tokens[$previousIndex] = new Token(array(T_WHITESPACE, str_repeat("\n", $expected)));
             }
+        } elseif (0 < $expected && $previous->isGivenKind(T_OPEN_TAG)) {
```

Comment:
I would not use Yoda style for this kind of comparison.

Reply:
we do it in rest of the repo, let it stay

Reply:
we don't enforce it in our `.php_cs.dist`, should I enable the option of YodaFixer to start checking for it?

Reply:
sure